iceoryx_platform/win/include/iceoryx_platform/windows.hpp

```diff
@@ -20,8 +20,9 @@
 #define _WINSOCKAPI_
 #define WIN32_LEAN_AND_MEAN
 
+#include <winsock2.h> // needs to be included before windows.h and therefore the blank line to prevent clang format from reordering
```

Comment:
I have to further investigate this. Although the header is now included before `windows.h` I still get a warning